### DISCUSSION TITLE
Not recognizing downloaded data - Formula 1 config

### DISCUSSION THREAD

**Discussion by acrane1** [Flexget/Flexget#3395] [General]
I'm trying to setup Flexget to download Formula 1 races for me. the way tv databases show the races, makes it so downloaders like Sonarr don't work. My work around is to use flexget and just do regex searches for what I want. The issue is it's downloading every version uploaded to the RSS feed. So it will download the 1080p race that I want, but when another 1080 version gets uploaded it'll download that too.
I tried using the series plugin, but once it downloads a practice session, or qualifying session it thinks that it's gotten all the files, so it will reject everything else.
Any suggestions?
Snippet of my config
`templates:
  qbittorrent:
    qbittorrent:
      path: /downloads_main/Formula1
      label: formula1
      host: ********
      port: *****
      tags:
        - flexget

  existing_files:
    exists:
        - /media
        - /tempfiles
        - /tempfiles2

tasks:
    formula1_races:
      rss:
        url: 'MY_RSS_FEED' 
      regexp:
        accept:
          - Formula1.*Race*
          - "Formula 1.*Race*"
        reject:
          - analysis
          - highlights
          - buildup
          - Notebook
          - practice
          - qualifying
          - sprint
          - mp4
        from: title
      quality: 1080p+
      template:
        - qbittorrent
        - existing_files`

**Comment by v-milenin** [comment]
Hi. I think, that i have the similar problem (https://github.com/Flexget/Flexget/discussions/3387), but i have only one download link to the same file, that updates with the new episode. And flexget didn't want download it more than once.

For now, i'm using the series plugin, that finds the tvshow in the RSS file and downloads it. Also i use the simple script that make flexget forget about this download, so when flexget will see the next time with tvshow in RSS, it downloads the torrent again, check the files on the disks, finds previosly downloaded series and downloads only the new one. It's a little uncomfortable, because i shouldn't delete already seen series, to not to download them again with the new one, but it works.
I use "/usr/local/bin/flexget series forget Lost" (for "lost" tvshow).
As an example: you have downloaded the practice, than you make flexget to "forget" it, and it will download qualifying next time it will see it in rss.

And in your case, I think, it's possible to find the way how to set regexps to accept and decline the races that you need.
With an examples of problem rss it will be better to understand the problem.
